package.json

```diff
@@ -1,82 +1,106 @@
-        "regression-tests": "node test/regression-tests.js",
-        "tests": "npm run unit-tests && npm run regression-tests",
-
-        "analyze-coverage": "istanbul cover test/unit-tests.js",
-        "check-coverage": "istanbul check-coverage --statement 100 --branch 100 --function 100",
-        "dynamic-analysis": "npm run analyze-coverage && npm run check-coverage",
-
-        "test": "npm run tests && npm run static-analysis && npm run dynamic-analysis",
-
-        "profile": "node --prof test/profile.js && mv isolate*.log v8.log && node-tick-processor",
-        "benchmark": "node test/benchmarks.js",
-        "benchmark-quick": "node test/benchmarks.js quick",
-
-        "coveralls": "coveralls < ./coverage/lcov.info",
-        "downstream": "node test/downstream.js",
-        "travis": "npm test && npm run coveralls && npm run downstream",
-
-        "generate-regex": "node tools/generate-identifier-regex.js"
+  "name": "esprima",
+  "description": "ECMAScript parsing infrastructure for multipurpose analysis",
+  "homepage": "http://esprima.org",
+  "main": "esprima.js",
+  "bin": {
+    "esparse": "./bin/esparse.js",
+    "esvalidate": "./bin/esvalidate.js"
+  },
+  "version": "2.4.0",
+  "files": [
+    "bin",
+    "unit-tests.js",
+    "esprima.js"
+  ],
+  "engines": {
+    "node": ">=0.10.0"
+  },
+  "author": {
+    "name": "Ariya Hidayat",
+    "email": "ariya.hidayat@gmail.com"
+  },
+  "maintainers": [
+    {
+      "name": "Ariya Hidayat",
+      "email": "ariya.hidayat@gmail.com",
+      "web": "http://ariya.ofilabs.com"
     }
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jquery/esprima.git"
+  },
+  "bugs": {
+    "url": "https://github.com/jquery/esprima/issues"
+  },
+  "license": "BSD-2-Clause",
+  "devDependencies": {
+    "chai": "~3.0.0",
+    "coveralls": "~2.11.2",
+    "escomplex-js": "1.2.0",
+    "eslint": "~0.23.0",
+    "gulp": "^3.9.0",
+    "gulp-concat": "^2.6.0",
+    "gulp-karma": "0.0.4",
+    "gulp-sequence": "^0.4.0",
+    "gulp-watch": "^4.3.2",
+    "istanbul": "~0.3.16",
+    "jscs": "~1.13.1",
+    "json-diff": "~0.3.1",
+    "karma": "~0.12.37",
+    "karma-chai": "~0.1.0",
+    "karma-chrome-launcher": "~0.2.0",
+    "karma-cli": "~0.1.0",
+    "karma-coverage": "~0.4.2",
+    "karma-firefox-launcher": "~0.1.6",
+    "karma-leche": "0.0.0",
+    "karma-mocha": "~0.2.0",
+    "karma-phantomjs-launcher": "~0.2.0",
+    "karma-sinon": "~1.0.4",
+    "karma-sourcemap-loader": "^0.3.5",
+    "karma-webpack": "^1.6.0",
+    "leche": "^2.1.1",
+    "lodash": "~3.10.0",
+    "map-stream": "0.0.6",
+    "mocha": "^2.2.5",
+    "node-libs-browser": "^0.5.2",
+    "node-tick-processor": "~0.0.2",
+    "phantomjs": "~1.9.17",
+    "regenerate": "~0.6.2",
+    "sinon": "~1.15.4",
+    "unicode-7.0.0": "~0.1.5",
+    "vinyl": "^0.5.0",
+    "vinyl-fs": "^1.0.0",
+    "vinyl-source-stream": "^1.1.0",
+    "webpack": "^1.10.1"
+  },
```

Comment:
there are probably several libraries we're no longer using, but i added while developing